pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -64,7 +64,7 @@ var _ PodContainerManager = &podContainerManagerImpl{}
 func (m *podContainerManagerImpl) Exists(pod *v1.Pod) bool {
 	podContainerName, _ := m.GetPodContainerName(pod)
 	if err := m.cgroupManager.Validate(podContainerName); err != nil {
-		klog.V(4).InfoS("Validate cgroup error", "podContainerName", podContainerName, "err", err)
+		klog.V(4).InfoS("Validate cgroup error", "cgroupName", podContainerName, "err", err)
```

Comment:
/lgtm

Reply:
please, rebase your commits into one, thanks
/hold

Reply:
/label tide/merge-method-squash

Reply:
@lixiv Still not squashed. Can you squash them locally and push?

Reply:
/lgtm cancel